src/components/MessageReactions/MessageReactions.tsx

```diff
@@ -17,8 +30,76 @@ const MessageReactions = () => {
     channel.publish('send', message)
   }
 
+  const sendMessageReaction = (emoji: string, timeserial: any) => {
+    channel.publish('reaction', {
+      body: emoji,
+      extras: {
+        reference: { type: 'com.ably.reaction', timeserial: timeserial },
+      },
+    })
+    setShowEmojiList(false)
+  }
+
+  const getMessageReactions = () => {
+    channel.subscribe(
+      {
+        name: 'reaction',
+        refTimeserial: chatMessage.timeserial,
+      },
+      (reaction) => {
+        // Update current chat with its reactions
+        if (!msgReactions.includes(reaction.data.body)) {
+          msgReactions.push(reaction.data.body)
+        }
+        setChatMessage((chatMessage) => ({
+          ...chatMessage,
+          reactions: msgReactions,
+        }))
+      }
+    )
+  }
+
+  const updateMessageFromHistory = (
+    messageIndex: number,
+    history: Types.PaginatedResult<Types.Message>
+  ) => {
+    const lastPublishedMessage = history?.items[messageIndex]
+    // Get reactions of the published message
+    if (messageIndex > 0) {
+      for (let i = messageIndex - 1; i >= 0; i--) {
+        if (!msgReactions.includes(history?.items[i].data.body)) {
+          msgReactions.push(history?.items[i].data.body)
+        }
+      }
+    }
+    // Update chat message
+    setChatMessage({
+      author: lastPublishedMessage?.data.author,
+      content: lastPublishedMessage?.data.content,
+      timeserial: lastPublishedMessage?.extras.timeserial,
+      reactions: msgReactions,
+    })
+  }
+
   useEffect(() => {
-    sendMessage()
+    // Subscribe to message reactions
+    getMessageReactions()
+    // Keep last published message and reactions
+    channel.history((err, result) => {
+      // Get index of last sent message from history
+      const lastPublishedMessageIndex: any = result?.items.findIndex(
+        (message) => {
+          return message.name == 'send'
+        }
+      )
```

Comment:
Since this is a single line with a return statement you can simply it to
```suggestion
      const lastPublishedMessageIndex: any = result?.items.findIndex((message) => message.name == 'send')
```